### PR TITLE
[enterprise-4.14] Telcodocs-1511-414 (TELCODOCS-1907) CNF-8804 Tuning CNI- ALLMULTI flag (GA, 4.14.z backport)

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-1511,[TELCODOCS-1907](https://issues.redhat.com//browse/TELCODOCS-1907)]: Enabling ALLMULTI flag

Version(s): 4.14

Issue: https://issues.redhat.com/browse/TELCODOCS-1907

Link to docs preview: 

- https://80042--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-interface-sysctl-sriov-device.html#enabling-all-multicast-sriov-network_configuring-sysctl-interface-sriov-device
- https://80042--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/hardware_networks/configuring-interface-sysctl-sriov-device.html#nw-about-all-multi-cast-mode_configuring-sysctl-interface-sriov-device
- https://80042--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/configure-syscontrols-interface-tuning-cni

QE review:
- [x] QE has approved this change.

Additional information: “Cherry Picked from https://github.com/openshift/openshift-docs/commit/ce719b38d7f50f786bb084d72e2f61a2448ffef6 xref: [https://github.com/openshift/openshift-docs/pull/68346/]

### CODE REVIEW

**Review by tmalove — APPROVED** [review]

**Comment by adellape** [comment]
@kquinn1204 Sorry, I saw this in the merge-review queue but it looks like it was added to the queue before the peer-review was completed. Tami has finished her peer review now, so please review her comments and then re-submit for merge-review when ready. Thank you! Dropping merge-review labels for now.

File: modules/nw-cfg-tuning-interface-cni.adoc [hunk 1]
```diff
@@ -3,9 +3,9 @@
 // * networking/setting-interface-level-network-sysctls.adoc
 :_mod-docs-content-type: PROCEDURE
 [id="nw-configuring-tuning-cni_{context}"]
-= Configuring the tuning CNI
+= Configuring system controls by using the tuning CNI
 
-The following procedure configures the tuning CNI to change the interface-level network `net.ipv4.conf.IFNAME.accept_redirects` sysctl. This example enables accepting and sending ICMP-redirected packets.
+The following procedure configures the tuning CNI to change the interface-level network `net.ipv4.conf.IFNAME.accept_redirects` sysctl. This example enables accepting and sending ICMP-redirected packets. In the tuning CNI meta plugin configuration, the interface name is represented by the `IFNAME` token and is replaced with the actual name of the interface at runtime.
```
[the remark this answers is not in the document]
Comment (kquinn1204) [reply modules/nw-cfg-tuning-interface-cni.adoc:8]: Based on previous discussion in vale meeting I believe plugin is acceptable and what we use throughout the OpenShift docs

File: modules/nw-sriov-about-all-multi-cast-mode.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+//
+// * networking/hardware_networks/configuring-interface-sysctl-sriov-device.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-about-all-multi-cast-mode_{context}"]
+= About all-multicast mode
+
+Enabling all-multicast mode, particularly in the context of rootless applications, is critical. If you do not enable this mode, you would be required to grant the `NET_ADMIN` capability to the pod's Security Context Constraints (SCC). If you were to allow the `NET_ADMIN` capability to grant the pod privileges to make changes that extend beyond its specific requirements, you could potentially expose security vulnerabilities.
+
+The tuning CNI plugin supports changing several interface attributes, including all-multicast mode. By enabling this mode, you can allow applications running on Virtual Functions (VFs) that are configured on a SR-IOV network device to receive multicast traffic from applications on other VFs, whether attached to the same or different physical functions.
```
[the remark this answers is not in the document]
Comment (kquinn1204) [reply modules/nw-sriov-about-all-multi-cast-mode.adoc:11]: plugin acceptable and used throughout OCP docs

File: networking/hardware_networks/configuring-interface-sysctl-sriov-device.adoc [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 [id="configuring-interface-level-sysctl-settings-sriov-device"]
-= Configuring interface-level network sysctl settings for SR-IOV networks
+= Configuring interface-level network sysctl settings and all-multicast mode for SR-IOV networks
 include::_attributes/common-attributes.adoc[]
 :context: configuring-sysctl-interface-sriov-device
 
 toc::[]
 
-As a cluster administrator, you can modify interface-level network sysctls using the tuning Container Network Interface (CNI) meta plugin for a pod connected to a SR-IOV network device.
+As a cluster administrator, you can change interface-level network sysctls and several interface attributes such as promiscuous mode, all-multicast mode, MTU, and MAC address by using the tuning Container Network Interface (CNI) meta plugin for a pod connected to a SR-IOV network device.
```
[the remark this answers is not in the document]
Comment (kquinn1204) [reply networking/hardware_networks/configuring-interface-sysctl-sriov-device.adoc:9]: plugin agreed in vale and used in OCP

**Review by adellape — COMMENTED** [review]
Holding merge mainly for the `.Procedure` and `[source,terminal]` comment.

**Comment by kquinn1204** [comment]
@adellape this PR has already been notified to CM so good to merge

**Review by adellape — APPROVED** [review]

**Pull request merged**